changedetectionio/blueprint/tags/templates/edit-tag.html

```diff
@@ -69,9 +69,9 @@
                                 {% endif %}
                             </ul>
                         </li>
-                        <li>XPath - Limit text to this XPath rule, simply start with a forward-slash,
+                        <li>XPath - Limit text to this XPath rule, simply start with a forward-slash. To specify XPath to be used explicitly or the XPath rule starts with an XPath function: Prefix with <code>xpath:</code>
                             <ul>
-                                <li>Example:  <code>//*[contains(@class, 'sametext')]</code> or <code>xpath://*[contains(@class, 'sametext')]</code>, <a
+                                <li>Example:  <code>//*[contains(@class, 'sametext')]</code> or <code>xpath:count(//*[contains(@class, 'sametext')])</code>, <a
```

Comment:
what was the reason for this addition? does `xpath:` also do 3.0 but // does 1.0? or?

Reply:
count() function is for 2.0-3.1

Reply:
This is the reason(https://github.com/dgtlmoon/changedetection.io/pull/1774/files#r1320659214).